scripts/dockerimages/scripts/console.sh

```diff
@@ -74,6 +74,18 @@ EOF
 if ! grep -q '^UseDNS no' /etc/ssh/sshd_config; then
     echo "UseDNS no" >> /etc/ssh/sshd_config
 fi
+cat > /etc/os-release << EOF
```

Comment:
This same script is used for ubuntu.  You need to detect if it's Ubuntu and change the ID_LIKE.  Basically I'd read the existing /etc/os-release and use the ID_LIKE from there.